Dockerfile

```diff
@@ -1,2 +1 @@
-ARG project_name=terra-dev-site
-FROM cerner/terra-node
+FROM mjhenkes/terra-node-ci
```

Comment:
I know we are swapping the base image out for alpine, but should our consumers feel this affect or care? Can we bump the terra-node version to v2 to use alpine and anyone pulling latest can get these changes without changing the image?

also I assume terra is the only one using this 😄

Reply:
Consumers don't care, we just never versioned terra-node and I didn't want to have to go and fix all our repos at once. All the other repos are just pulling in terra-node latest and would have all broken.